Error page

ebradd5 opened the pull request.

Dorugoramon committed:
Added error handling functionality in app.py

Dorugoramon committed:
added routes for 400,404, and 500 errors

Dorugoramon committed:
Merge branch 'main' into errorhandling

Dorugoramon committed:
Delete flask_app.py

Dorugoramon committed:
Update error400.html

Dorugoramon committed:
Update error404.html

Dorugoramon committed:
Update error500.html

Dorugoramon committed:
Update errorpage.html

ebradd5 committed:
pages for errors

ebradd5 committed:
fixed errorpage.htm to match other error pages

Dorugoramon commented on templates/error400.html:
```diff
@@ -0,0 +1,22 @@
+{% include 'header.html' %}
```
I dont know how to do that,

Dorugoramon reviewed (commented):
Looks good to me.

SeanDonnelly1 committed:
Removed error routes

SeanDonnelly1 committed:
Rework of error page

Updated code to use one universal error page instead of separate ones for each error

SeanDonnelly1 committed:
Fixed typo

SeanDonnelly1 committed:
Modified error messages and deleted unused files
